packages/trader/src/Modules/Trading/Components/Elements/PayoutPerPoint/payout-per-point-input.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import WheelPicker from '../../Form/WheelPicker';
+import { Popover, Text, useDevice } from '@deriv/components';
+import Fieldset from 'App/Components/Form/fieldset';
+import { Localize, localize } from '@deriv/translations';
+import './payout-per-point.scss';
+import { LabelPairedChevronsDownCaptionRegularIcon, LabelPairedChevronsUpCaptionRegularIcon } from '@deriv/quill-icons';
+
+const PayoutPerPointInput = ({
+    payoutOptions,
+    onPayoutClick,
+    selectedBarrier,
+    defaultPayout,
+    currency,
+    tooltipText,
+}: {
+    payoutOptions: number[];
+    onPayoutClick: (option: number) => void;
+    selectedBarrier: string;
+    defaultPayout: number;
+    currency: string;
+    tooltipText?: React.ReactNode;
+}) => {
+    const { is_desktop } = useDevice();
+    const turbos_payout_message = (
+        <Localize i18n_default_text='The ammount you will receive at expiry for every point of change above the barrier.' />
+    );
+    return is_desktop ? (
+        <Fieldset
+            className={'trade-container__fieldset payout-per-point-input'}
+            header={localize('Payout per Point')}
```

Comment:
Seems that `header` accepts `ReactNode`, might it be better to pass `<Localize/>`?